frontend/src/components/ParticlesComponent/ParticlesComponent.scss

```diff
@@ -0,0 +1,3 @@
+canvas {
+  height: 100vh !important;
```

Comment:
Usually `!important` is a red flag. This should be avoided at all costs. Why did you add this here?